doc/reference_manual/tutorial_new.tex

```diff
@@ -0,0 +1,20 @@
+\section{Tutorials}
```

Comment:
Should be rather chapter. Make note in introductory example about further tutorials in this chapter.  Move tutorials after description of models and file formats, just before generated input reference.